src/Fantomas.Core.Tests/ListTests.fs

```diff
@@ -2319,3 +2319,22 @@ let x: double[][] = [ [ 1.0, 2.0, 3.0 ] ]
 let foo (x: double[]) (y: object[][]) : string[,] = x :> int[]
 let foo<'T> (x: 'T[]) = x
 """
+
+[<Test>]
+let ``Maintain indentation on function list, 2158`` () =
```

Comment:
Could you lower-case `Maintain` please?
https://github.com/fsprojects/fantomas/blob/master/CONTRIBUTING.md#unit-test